.kiro/steering/tech.md

```
@@ -1,108 +1,256 @@
-- **CDN**: GitHub Pages provides global CDN
-
-## Future Tech Stack (Planned)
-- **React 18+**: UI framework migration (Phase 5)
-- **React-Three-Fiber**: React renderer for Three.js
-- **React-Drei**: Three.js helpers for R3F
-- **Miniplex-React**: React integration for ECS
-- **Zustand**: State management
\ No newline at end of file
+## CI/CD Pipeline
+
+### GitHub Actions Workflow
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)